doc/README.md

```diff
@@ -35,6 +35,7 @@
 * [Changelog](../CHANGELOG.md)
 * [Benchmark](benchmark.md)
 * [Code Style](../CODE_STYLE.md)
+* [Debug function](debugging.md)
```

Comment:
I do not think `debugging.md` is a good name for this doc

Reply:
updated.